autogalaxy/plot/plotters/hyper_plotters.py

```diff
@@ -0,0 +1,151 @@
+from autoarray.plot.plotters import abstract_plotters
+from autogalaxy.plot.mat_wrap import lensing_mat_plot, lensing_include, lensing_visuals
+
+
+class HyperPlotter(abstract_plotters.AbstractPlotter):
+    def __init__(
+        self,
+        mat_plot_2d: lensing_mat_plot.MatPlot2D = lensing_mat_plot.MatPlot2D(),
+        visuals_2d: lensing_visuals.Visuals2D = lensing_visuals.Visuals2D(),
+        include_2d: lensing_include.Include2D = lensing_include.Include2D(),
+    ):
+        super().__init__(
+            mat_plot_2d=mat_plot_2d, include_2d=include_2d, visuals_2d=visuals_2d
+        )
+
+    @property
+    def visuals_with_include_2d(self) -> lensing_visuals.Visuals2D:
+        """
+        Extracts from a `Structure` attributes that can be plotted and return them in a `Visuals` object.
+
+        Only attributes with `True` entries in the `Include` object are extracted for plotting.
+
+        From an `AbstractStructure` the following attributes can be extracted for plotting:
+
+        - origin: the (y,x) origin of the structure's coordinate system.
+        - mask: the mask of the structure.
+        - border: the border of the structure's mask.
+
+        Parameters
+        ----------
+        structure : abstract_structure.AbstractStructure
+            The structure whose attributes are extracted for plotting.
+
+        Returns
+        -------
+        vis.Visuals2D
+            The collection of attributes that can be plotted by a `Plotter2D` object.
+        """
+        return self.visuals_2d
+
+    @abstract_plotters.for_figure
+    def figure_hyper_model_image(self, hyper_model_image):
+        """Plot the image of a hyper_galaxies galaxy image.
+
+        Set *autogalaxy.datas.arrays.mat_plot_2d.mat_plot_2d* for a description of all input parameters not described below.
+
+        Parameters
+        -----------
+        hyper_galaxy_image : datas.imaging.datas.Imaging
+            The hyper_galaxies galaxy image.
+        origin : True
+            If true, the origin of the datas's coordinate system is plotted as a 'x'.
+        """
+        self.mat_plot_2d.plot_array(
+            array=hyper_model_image, visuals_2d=self.visuals_with_include_2d
+        )
+
+    @abstract_plotters.for_figure
+    def figure_hyper_galaxy_image(self, galaxy_image):
+        """Plot the image of a hyper_galaxies galaxy image.
+
+        Set *autogalaxy.datas.arrays.mat_plot_2d.mat_plot_2d* for a description of all input parameters not described below.
+
+        Parameters
+        -----------
+        hyper_galaxy_image : datas.imaging.datas.Imaging
+            The hyper_galaxies galaxy image.
+        origin : True

```

Comment:
All these functions seem to be the same but they take a different argument. Why not just have one function?